### PR TITLE
New term entry for Java Math.negateExact()

### LINKED ISSUE #905: [Term Entry] Java Math Methods: `.negateExact()`
### Reason/inspiration (optional)

Under `java/concepts/math-methods/terms/`, create a new term entry for the `Math.negateExact()` Java method.

- In the `terms/` folder, make a new folder called `negate-exact`
- Inside the `negate-exact` folder, create a new markdown file called `negate-exact.md`.
- Inside the file, be sure to include a "Syntax" section with a \```pseudo code snippet.
- Also make sure to include an "Example" section with a \```java code snippet.

Please refer to the [entry templates](https://github.com/Codecademy/docs/blob/main/documentation/), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md), and [markdown style guide](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/) when working on the PR for this issue.

### Entry Type (select all that apply)

New Term for Existing Concept Entry

### Code of Conduct

- [X] I agree to follow Codecademy Doc's Code of Conduct

### For Maintainers

- [x] Labels added
- [x] Issue is assigned

### PR DESCRIPTION
**Pull request by YanisaHS**
### Description

I added the .negateExact() Java Math method in java/concepts/math-methods/terms/. The file:

- Defines .negateExact()
- Provides the syntax
- Provides an example
Closes #905 

### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/java/concepts/math-methods/terms/negate-exact/negate-exact.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+  - 'Arithmetic'
+  - 'Functions'
+  - 'Methods'
+  - 'Numbers'
+CatalogContent:
+  - 'learn-java'
+  - 'paths/computer-science'
+---
+
+The `Math.negateExact()` method returns the negation of the argument and raises an [exception](https://www.codecademy.com/resources/docs/java/errors) if the result overflows.
```
Comment (Dusch4593) [inline content/java/concepts/math-methods/terms/negate-exact/negate-exact.md:16]:
```suggestion
The **`Math.negateExact()`** method returns the negation of the argument and raises an [exception](https://www.codecademy.com/resources/docs/java/errors) if the result overflows.
```

File: content/java/concepts/math-methods/terms/negate-exact/negate-exact.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+    long num_long = 123L;
+    System.out.println(Math.negateExact(num_int));
+    System.out.println(Math.negateExact(num_long));
+  }
+}
+```
+
+This results in the following output:
+
+```java
```
Comment (Dusch4593) [inline content/java/concepts/math-methods/terms/negate-exact/negate-exact.md:44]:
```suggestion
```shell
```

File: content/java/concepts/math-methods/terms/negate-exact/negate-exact.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+
+The `Math.negateExact()` method returns the negation of the argument and raises an [exception](https://www.codecademy.com/resources/docs/java/errors) if the result overflows.
+
+## Syntax
+
+```pseudo
+Math.negateExact(num)
+```
+
+The `num` argument can be of type `int` or `long`.
```
Comment (Dusch4593) [inline content/java/concepts/math-methods/terms/negate-exact/negate-exact.md:24]:
```suggestion
The `num` parameter can be of type `int` or `long`.
```

In the context of describing the makeup of a method/function, let's refer to any passed values as parameters and as arguments in examples. 😄

**Review by Dusch4593 — CHANGES_REQUESTED** [review]
Hey @YanisaHS! Thanks for creating this new entry for Docs! 🎉

I've finished my review and left you some comments/suggestions. Can we also rename the folder and file name to `negateExact` and `negateExact.md`, respectively?

**Review by Dusch4593 — APPROVED** [review]
@YanisaHS LGTM for the next review 👍🏻

**Review by HishamT — APPROVED** [review]
LGTM!

**Comment by Dusch4593** [comment]
@YanisaHS LGTM for merge! 🚀

I made a small edit in 1ebb71303c8bd7e59a5eb35175978da5e8903e9e on line # 28.

**Comment by Dusch4593** [comment]
Thank you for another solid contribution to Codecademy Docs, @YanisaHS! 🥳

The new entry should now be live: https://www.codecademy.com/resources/docs/java/math-methods/negateExact

**Pull request merged**